forch/faucetizer.py

```diff
@@ -40,6 +40,10 @@ def process_device_behavior(self, eth_src, behavior):
             device = self._devices.setdefault(eth_src, Device())
             device.behavior.CopyFrom(behavior)
 
+    def process_segments_to_vlans(self, segments_to_vlans):
+        """Process single segment vlan mapping"""
```

Comment:
if it's single segment, then it should be segment_to_vlan (not plurals). Where is this function used? I don't see it anywhere in the PR so I can't tell what the intended use is.